yt/frontends/ramses/io.py

```diff
@@ -189,12 +232,29 @@ def _read_particle_fields(self, chunks, ptf, selector):
         fields = [
             (ptype, fname) for ptype, field_list in ptf.items() for fname in field_list
         ]
+
         for ptype, field_list in sorted(ptf.items()):
             for ax in "xyz":
                 if pn % ax not in field_list:
                     fields.append((ptype, pn % ax))
-        for chunk in chunks:
-            for subset in chunk.objs:
+
+            if ptype != "sink_csv":
```

Comment:
I'm confused. Shouldn't the guard clause be `if ptype == "sink_csv":` here ?

Reply:
There is an if and then an else, so it's a bit equivalent, isn't it  ?

I was to force to add this condition as the reader is really not the same between the fortran file and the csv file. I didn't manage to find a way without.

Reply:
Indeed there's an `else` in there, and having this `if/else` branching isn't an issue, but I would expect the *old* code to correspond to previously supported cases (i.e. `if ptype != "sink_csv"`). Here, it seems to me that the *old* code is run specifically in the `else` clause, which matches the case you are adding. Am I missing anything ?

Reply:
ok I see. I will change with this idea